src/Face.ts

```diff
@@ -31,8 +24,8 @@ export class Face {
   area: number
 
   constructor() {
-    this.normal = [-1, -1, -1]
-    this.centroid = [-1, -1, -1]
+    this.normal = new Float32Array(3)
```

Comment:
Hmm.... I'm expecting some kind of performance impact from this. Typed arrays are expensive.

Reply:
I see, I'll revert to regular arrays, thanks for the suggestion.